include/exml.hrl

```diff
@@ -3,7 +3,7 @@
 
 -record(xmlcdata, {content = [] :: iodata()}).
 
--record(xmlel, {name :: binary(),
+-record(xmlel, {name :: iodata(),
```

Comment:
Since it's `binary()` in output anyway, let's stick to `binary()` spec so it won't mislead any developer. :)

Reply:
👍